src/libcollections/slice.rs

```diff
@@ -2127,22 +2032,6 @@ mod tests {
     }
 
     #[test]
-    fn test_partition() {
```

Comment:
These tests were actually duplicated in `vec.rs` for some reasons, but I kept those!